### PR TITLE
Release v0.12

### PR DESCRIPTION
**Pull request by jch**
cc @ruby-ldap/ruby-ldap-admins

### CODE REVIEW

File: lib/net/ldap/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Net
   class LDAP
-    VERSION = "0.11"
+    VERSION = "0.12"
```
Comment (satoryu) [inline lib/net/ldap/version.rb]: Why isn't [Semantic Versioning](http://guides.rubygems.org/patterns/#semantic-versioning) adopted?

Comment (jch) [inline lib/net/ldap/version.rb]: @satoryu do you just mean why we're missing the patch number?

Comment (satoryu) [inline lib/net/ldap/version.rb]: Yes.

Comment (jch) [inline lib/net/ldap/version.rb]: Fixed in 98d122d3f94707e2367da8353ee2610ec095ff7c. Thanks!

Comment (satoryu) [inline lib/net/ldap/version.rb]: [LGTM](http://lgtm.in/i/SxXXctwTp)

**Comment by jch** [comment]
@satoryu thanks for the review!

**Pull request merged**